### DISCUSSION TITLE
Insane Amount of RAM usage after a while of Playing

### DISCUSSION THREAD

**Discussion by blackreapersora** [ppy/osu#23341] [Q&A] [answered]
Has been like this for ages, never really thought about making a post here since I thought it would get fixed eventually, but It didnt which leads me to think that it might be a Local issue which im not sure of tho. Cause of this *might* be swapping skins repedeately/often, since the ram usage goes up the more I play (there was a RAM usage issue with beatmaps not unloading a while ago(#2787) thats why I think its the skins in this case).

**Comment by peppy** [comment, marked as answer]
You're probably using the opengl renderer. Please switch to legacy opengl.

**Reply by blackreapersora** [reply]
I use legacy because the new one doesnt run well on my pc for some reason. I can send logs and specs if necessary
